### ISSUE TITLE
Please explain the purpose

### ISSUE THREAD

**Issue by philbowles** [martinloren/ESP32-Serial-Debugger#1]
What am I missing? "and visualize the app on with a Web" what app ?  the app on the esp32 or an app on the phone? what is being visualised? Where does "serial" come in - this seems to be just a webserver on an esp32 - what does the esp32 DO? How about some screenshots of the webserver to give us a clue? 

Your documentation explains nothing and is totally baffling - what is this actually for? What purpose does it serve? Who would use it why?

**Comment by martinloren** [comment]
You are right, I've added a screenshots and some more info, thanks for the suggestions!
